test/mpi/coll/testlist.in

```diff
@@ -56,6 +56,8 @@ coll10 4
 coll11 4
 coll12 4
 coll13 4
+customreduce 1
```

Comment:
I think you would want to test it for at least 2 ranks?

Reply:
The tests fail for one. I will add two ranks, but the issue isn't with ranks.

Reply:
Does it fail only when number of ranks is 1? With 1 rank, reduce and allreduce should return immediately without having to do any computation/communication.

Reply:
The tests fail with ranks>=1.

Reply:
Any more information about where the failure occurs in MPICH for this test? Some additional information would be helpful, either in the commit message or in the test comments.